homeassistant/components/lutron_caseta/binary_sensor.py

```diff
@@ -56,6 +56,15 @@ def unique_id(self):
         """Return a unique identifier."""
         return f"occupancygroup_{self.device_id}"
 
+    @property
+    def device_info(self):
+        """Return the device info.
+
+        Sensor entities are aggregated from one or more physical
+        sensors by each room. Therefore, there shouldn't be devices
+        related to any sensor entities.
+        """
```

Comment:
We should `return None` here explicitly even though pylint doesn't understand that. We can disable pylint locally.
```suggestion
    @property
    def device_info(self):
        """Return the device info.

        Sensor entities are aggregated from one or more physical
        sensors by each room. Therefore, there shouldn't be devices
        related to any sensor entities.
        """
        return None  # pylint: disable=useless-return
```